HPXMLtoOpenStudio/measure.rb

```diff
@@ -1614,26 +1614,24 @@ def self.add_cooling_system(runner, model, spaces, airloop_map)
       end
 
       sys_id = cooling_system.id
-      if [HPXML::HVACTypeCentralAirConditioner].include? cooling_system.cooling_system_type
-
-        airloop_map[sys_id] = HVAC.apply_central_air_conditioner_furnace(model, runner, cooling_system, heating_system,
-                                                                         sequential_cool_load_fracs, sequential_heat_load_fracs,
-                                                                         living_zone)
-
-      elsif [HPXML::HVACTypeRoomAirConditioner].include? cooling_system.cooling_system_type
-
-        HVAC.apply_room_air_conditioner(model, runner, cooling_system,
-                                        sequential_cool_load_fracs, living_zone)
+      if [HPXML::HVACTypeCentralAirConditioner,
+          HPXML::HVACTypeRoomAirConditioner,
+          HPXML::HVACTypeMiniSplitAirConditioner,
+          HPXML::HVACTypePTAC].include? cooling_system.cooling_system_type
+        if cooling_system.cooling_system_type == HPXML::HVACTypeRoomAirConditioner
+          HVAC.apply_central_air_source_hvac_systems(model, runner, cooling_system, heating_system,
+                                                     sequential_cool_load_fracs, sequential_heat_load_fracs,
+                                                     living_zone)
+        else
+          airloop_map[sys_id] = HVAC.apply_central_air_source_hvac_systems(model, runner, cooling_system, heating_system,
+                                                                           sequential_cool_load_fracs, sequential_heat_load_fracs,
+                                                                           living_zone)
+        end
```

Comment:
@shorowit Just ran a quick test for room ac, there's no difference creating an airloop or not looking at the simulation results. I think both ways have pros and cons, if we create airloop, then room ac doesn't need extra "if" to differentiate from other central air source hvac systems, if we eliminate airloop, it can make the model simpler (and maybe some performance improvement?).

Reply:
That's good to hear. I think you're right about the tradeoffs. I'm fine with what you have in the PR (simpler code, as opposed to a simpler model).

Reply:
There's no harm using a single call here if air loop is created for room ac, just verified that results are the same.